DEV-Simple/core/MediaManager.swift

```diff
@@ -67,6 +67,32 @@ class MediaManager: NSObject {
         avPlayer?.seek(to: CMTime(seconds: seconds, preferredTimescale: CMTimeScale(NSEC_PER_SEC)))
     }
 
+    private func seekForward(_ sender: Any) {
+        guard let duration  = avPlayer?.currentItem?.duration else {
+            return
+        }
+        let playerCurrentTime = CMTimeGetSeconds(avPlayer!.currentTime())
+        let newTime = playerCurrentTime + 15
+
+        if newTime < (CMTimeGetSeconds(duration) - 5) {
```

Comment:
Should this conditional check for `(CMTimeGetSeconds(duration) - 15` instead of `(CMTimeGetSeconds(duration) - 5`?